mu-plugins/blocks/sidebar-container/src/view.js

```diff
@@ -136,6 +136,31 @@ function init() {
 			container.classList.add( 'is-fixed-sidebar' );
 			onScroll(); // Run once to avoid footer collisions on load (ex, when linked to #reply-title).
 			window.addEventListener( 'scroll', onScroll );
+
+			// eslint-disable-next-line no-undef
+			const observer = new ResizeObserver( () => {
+				// When the mainEl includes class "is-bottom-sidebar" and undergoes resizes,
+				// it means that the sidebar is positioned at its bottommost location.
+				// In this case, It is necessary to manually trigger the resize
+				// under the condition (footerStart - gap < sidebarBottom) to adjust the position appropriately.
+				// Otherwise, the sidebar will remain at the bottom of the page, leading to a broken page layout.
```

Comment:
```suggestion
				// If the sidebar is positioned at the bottom and mainEl resizes,
				// it will remain fixed at the previous bottom position, leading to a broken page layout.
				// In this case manually trigger the scroll handler to reposition.
```